src/libp2p/Directory.Packages.props

```diff
@@ -5,7 +5,7 @@
   <ItemGroup>
     <PackageVersion Include="BenchmarkDotNet" Version="0.13.11" />
     <PackageVersion Include="BenchmarkDotNet.Diagnostics.Windows" Version="0.13.11" />
-    <PackageVersion Include="BouncyCastle.Cryptography" Version="2.2.1" />
+    <PackageVersion Include="BouncyCastle.Cryptography" Version="2.3.1" />
```

Comment:
Why not the latest version?

Reply:
Just to make it the same as for other dependencies for now. We can upgrade it separately with packaging, wdyt?